### PR TITLE
Update configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by veeruphd**

### CODE REVIEW

**Comment by kundan2707** [comment]
Duplicate of #44077

**Comment by T-Lakshmi** [comment]
@veeruphd,
Thanks for your contribution.
It seems that you have opened two pull requests simultaneously, proposing the same changes in both. It is not permitted to open multiple pull requests with identical content.

I suggest you please close this PR and you may proceed with PR #44077.
To know more about k8s contribution, please read this [doc](https://www.kubernetes.dev/docs/guide/).

**Comment by veeruphd** [comment]
thank you laxmi 
Regardsveeresh
    On Monday, November 27, 2023 at 01:27:04 AM EST, lakshmi prasuna ***@***.***> wrote:  
 
 


@veeruphd,
Thanks for your contribution.
It seems that you have opened two pull requests simultaneously, proposing the same changes in both. It is not permitted to open multiple pull requests with identical content.

I suggest you please close this PR and you may proceed with PR #44077.
To know more about k8s contribution, please read this doc.

**Review by dipesh-rawat — COMMENTED** [review]
@veeruphd This PR would greatly benefit from including a description _[(click here to add details)](https://github.com/kubernetes/website/pull/44079#issue-2010389343)_ of the changes made, along with the reasons behind proposing them. This will not only improve clarity but also facilitate a more straightforward understanding and evaluation of your contribution by the reviewers.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
 
 Below is an example for taking a snapshot of the keyspace served by
 `$ENDPOINT` to the file `snapshot.db`:
+step1:Take an etcd snapshot backup using the following command
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:279]: I'd use an [ordered list](https://www.google.com/search?q=ordered+list+in+Markdown) here.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 2]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
...
+
+ETCDCTL_API=3 etcdctl \
+  --endpoints=https://127.0.0.1:2379 \
+  --cacert=<ca-file> \
+  --cert=<cert-file> \
+  --key=<key-file> \
+  snapshot save <backup-file-location>
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:286]: This should be a block of code - see https://www.google.com/search?q=fenced+code+block+in+Markdown

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 3]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
...
+
+  Step2 : How do we get the endpoint and the certificate informatio
+  we can retrieve them from the etcd pod and the manifest file for etcd pod is located under /etc/kubernetes/manifests folder
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:289]: I'd use an [ordered list](https://www.google.com/search?q=ordered+list+in+Markdown) item here.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 4]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
...
+  
+  cat /etc/kubernetes/manifests/etcd.yaml | grep listen
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:291]: This should be a block of code - see https://www.google.com/search?q=fenced+code+block+in+Markdown

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 5]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
...
+![image](https://github.com/kubernetes/website/assets/151970071/a8116d2f-d4b5-4207-a6fb-cd4508e71b41)
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:292]: This doesn't look right.

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 6]
```diff
@@ -276,6 +276,33 @@ not affect the performance of the member.
...
+
+  step 3:we can get the certificate information with the following command.
+  
+ cat /etc/kubernetes/manifests/etcd.yaml | grep file
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:296]: This should be a block of code - see https://www.google.com/search?q=fenced+code+block+in+Markdown

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:1]: As well as the inline feedback, please also [avoid using ”we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we).

**Review by sftim — COMMENTED** [review]
Thanks.

- Please set a PR title and description that outlines the change and summarizes why we should make it
- We'd like the changes to follow our [style guide](https://kubernetes.io/docs/contribute/style/style-guide/) for documentation

**Comment by natalisucks** [comment]
@veeruphd Hi there – do you have the bandwidth to address the feedback left by our other reviewers? Thanks!

**Comment by Ritikaa96** [comment]
Hi @veeruphd are you able to address the proposed reviews? If you arent actively working on this PR then you can mention it here and close it so that other contributor can pick it up.

**Pull request closed without merging**